app/src/main/java/io/github/droidkaigi/confsched2018/service/push/PushServiceModule.kt

```diff
@@ -0,0 +1,10 @@
+package io.github.droidkaigi.confsched2018.service.push
+
+import dagger.Binds
+import dagger.Module
+import dagger.android.AndroidInjector
+
+@Module
+interface PushServiceModule : AndroidInjector<PushService> {
+    @Binds fun providesService(service: PushService): PushService
```

Comment:
I think if you don't have PushService inject, PushServiceModule is not needed.
What do you think?

Reply:
I didn't know that! I will remove it :)